ietf/doc/views_draft.py

```diff
@@ -560,23 +560,22 @@ def to_iesg(request,name):
     if request.method == 'POST':
 
         if request.POST.get("confirm", ""): 
-
             by = request.user.person
 
             events = []
-
-            changes = []
-
-            if doc.get_state_slug("draft-iesg") == "idexists":
-                e = DocEvent()
-                e.type = "started_iesg_process"
-                e.by = by
-                e.doc = doc
-                e.rev = doc.rev
-                e.desc = "Document is now in IESG state <b>%s</b>" % target_state['iesg'].name
+            def append_event(type, desc):
+                e = DocEvent(type=type, by=by, doc=doc, rev=doc.rev, desc=desc)
```

Comment:
I worry a bit at taking values out of the parent scope this way. Lets either make all of these things arguments, or change this to a utility class where an instance gets its own copy of these things in its init.